packages/@dcl/sdk-commands/package.json

```diff
@@ -14,16 +14,17 @@
     "@dcl/mini-comms": "1.0.1-20230216163137.commit-a4c75be",
     "@dcl/protocol": "1.0.0-5555484353.commit-63c469d",
     "@dcl/rpc": "^1.1.1",
-    "@dcl/schemas": "^7.6.0",
+    "@dcl/schemas": "^8.2.1",
```

Comment:
I think you need to add the sepolia contracts and maybe some more config to the [linker-dapp](https://github.com/decentraland/linker-dapp) and bump it here too.